src/lcmap_client/cli.py

```diff
@@ -0,0 +1,103 @@
+import click
+import json
+import logging
+import numpy as np
+import pandas as pd
+
+from lcmap_client import Client
+
+log = logging.getLogger(__name__)
+
+spectra_to_bands = {
+  'blue':[
+    'LANDSAT_8/OLI_TIRS/sr_band2',
+    'LANDSAT_7/ETM/sr_band1',
+    'LANDSAT_5/TM/sr_band1'
+  ],
+  'green':[
+    'LANDSAT_8/OLI_TIRS/sr_band3',
+    'LANDSAT_7/ETM/sr_band2',
+    'LANDSAT_5/TM/sr_band2'
+  ],
+  'red':[
+    'LANDSAT_8/OLI_TIRS/sr_band4',
+    'LANDSAT_7/ETM/sr_band3',
+    'LANDSAT_5/TM/sr_band3'
+  ],
+  'ir':[
+    'LANDSAT_8/OLI_TIRS/sr_band5',
+    'LANDSAT_7/ETM/sr_band4',
+    'LANDSAT_5/TM/sr_band4'    
+  ],
+  'swir-1':[
+    'LANDSAT_8/OLI_TIRS/sr_band6',
+    'LANDSAT_7/ETM/sr_band5',
+    'LANDSAT_5/TM/sr_band5'
+  ],
+  'swir-2':[
+    'LANDSAT_8/OLI_TIRS/sr_band7',
+    'LANDSAT_7/ETM/sr_band7',
+    'LANDSAT_5/TM/sr_band7'
+  ],
+  'tirs-1':[
+    'LANDSAT_8/OLI_TIRS/toa_band10',
+    'LANDSAT_7/ETM/toa_band6',
+    'LANDSAT_5/TM/toa_band6'
+  ],
+  'cf':[
+    'LANDSAT_8/OLI_TIRS/cfmask',
+    'LANDSAT_7/ETM/cfmask',
+    'LANDSAT_5/TM/cfmask'
+  ]
+}
+
+
+@click.group()
+def cli():
+	pass
+
+
+@click.command()
+@click.option('--spectra', '-s', multiple=True, type=click.Choice('blue','green','red','ir','swir-1','swir-2','tirs-1','cf'))
+@click.option('--x', '-x', type=int)
+@click.option('--y', '-y', type=int)
+@click.option('--t1')
+@click.option('--t2')
+def rod(spectra, x, y, t1, t2):
+  client = Client()
+
+  if not spectra:
+    spectra = spectra_to_bands.keys()
+    print(spectra)
+  
+  result = []
```

Comment:
Heads-up: PEP-8 is 4-space indent. Google does 2-space (like Lisp). Let's stick with the PEP-8 way.